lib/tools/apk-utils.js

```diff
@@ -180,7 +181,8 @@ apkUtilsMethods.extractStringsFromApk = async function (apk, language, out) {
 
   try {
     log.debug("Reading strings from converted strings.json");
-    fileData = await fs.readFile(path.join(out, stringsJson), 'utf8');
+    localPath = path.join(out, stringsJson);
+    fileData = await fs.readFile(localPath, 'utf8');
```

Comment:
It looks like this is a local, home-grown `fs`. It would be good to shift to `appium-support`'s version.